spec/requests/users_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe "Users", type: :request do
+  describe "GET /index" do
+    it "returns http success" do
+      get users_path
+      expect(response).to have_http_status(:success)
```

Comment:
This spec is imcomplete as it's not verifying the behavior hapenning in the action implementation.